Marlin/src/feature/pause.cpp

```diff
@@ -93,16 +93,21 @@ static void ensure_safe_temperature() {
   }
 }
 
+void do_pause_e_move(const float &length, const float fr) {
```

Comment:
I suggest "do_pause_e_move" become "do_block_e_moves"
Because this code can be reused in firmware retract and many other , and for the future
with an external declaration in .h